backend/curation/api/curation-api.yml

```diff
@@ -14,6 +14,45 @@ info:
     ## Data Model
     <!-- Data model diagram created via Lucid: https://lucid.app/lucidchart/8d085832-34f4-4413-9c1d-44e77d06e326/edit?invitationId=inv_ed354532-7256-4e0a-9ad0-b622905e5198&page=Vfzkzdv~I1fL# -->
     <img src="/static/discover-api-data-model.png" width="50%"/>
+
+    ## Working with versions
+
+    #### Access to prior published version history
+    The CZ CELLxGENE Discover API employs the concept of *versions* for our two core entities, *Collections* and
+    *Datasets*. This is to permit access to prior published instances (versions) of these entities even as they are
+    changed over time via revisions (at present the Data Portal web application UI offers access to only the latest
+    published version of a given Collection and/or Dataset).
+
+    #### Version identifiers
+    While the *Collection* identifier `collection_id` will always refer to the latest published instance of a Collection
+    (like the "head" in version-control terminology), and must be used with `../collections/..` endpoints, the
+    *Collection version* identifier `collection_version_id` points always to one specific published instance (version)
+    of a Collection (like a commit hash) and must be used with `../collection_version/..`
+    endpoints. The same is true for *Datasets* and *Dataset versions*, though it is important to note that if a given Dataset
+    is not updated during the course of a revision of its parent Collection, then its *Dataset version* identifier
+    `dataset_version_id` will not change and will appear identically in the list of *Dataset versions* associated with the
+    Collection version from *before* the revision as well as with the Collection version from *after* the revision. Private Collections,
+    private revisions of published Collections, and their constituent (private) Datasets are not available as *versions*
+    until they are published, and therefore must be referenced via their `collection_id` and `dataset_id` identifiers
```

Comment:
I think it would be clearer if we defined where you can use version IDs instead of where you can't. Something like

 'version identifiers are used exclusively to fetch prior published versions of Collections and Datasets, via the collection_version and dataset_version endpoints. You may also fetch the metadata for all the prior published versions of a Collection or Dataset using collection_id or dataset_id using the '/versions' endpoints'

Reply:
I'm aligning the tags strictly with which endpoints use which type of identifier